src/main/java/com/minecolonies/coremod/client/gui/WindowBuildDecoration.java

```diff
@@ -130,20 +136,24 @@ public WindowBuildDecoration(final BlockPos pos, final String packMeta, final St
         blueprintFuture = StructurePacks.getBlueprintFuture(cleanedPackName, path);
         this.rotation = rotation;
         this.mirror = mirror;
+        this.buildRequestMessage = buildRequestMessage;
     }
 
-    @Override
-    public void onOpened()
+    private void confirmedBuild()
```

Comment:
Please dont reorganize existing methods in a PR, that makes reading it very hard and creates big conflict chunks